uniffi_bindgen/Cargo.toml

```diff
@@ -19,7 +19,7 @@ anyhow = "1"
 askama = { version = "0.11", default-features = false, features = ["config"] }
 cargo_metadata = "0.14"
 camino = "1.0.8"
-clap = { version = "~3.1", features = ["cargo", "std", "derive"] }
+clap = { version = "3.2.5", features = ["cargo", "std", "derive"] }
```

Comment:
Please stay at 3.1.
Fixing it to some higher version might be problematic when we integrate that back into mozilla-central, and I'd rather not deal with that at the moment.

Reply:
Is it okay for the minimum version to be 3.1, but allow higher versions within 3.x? We would ideally want to use 3.2 and also the `builtin-bindgen` feature. (at least we activate it now, I'm not 100% certain we actually need to)

Reply:
3.1 is fine, that what we use currently.
Shouldn't just `version = "3.1."` work now and then pick 3.2 for those that can and 3.1 for when they can't?

Reply:
Yes, that's what I meant with the minimum version. Will adjust.